myRental/test/test.js

```diff
@@ -0,0 +1,52 @@
+var assert = require('assert');
+var MockExpress = require('mock-express');
+var mysql = require("mysql");
+var fs = require('fs');
+var bcrypt = require('bcryptjs');
+var sys = require('sys');
+var exec = require('child_process').exec;
+
+var DataBase = require('../modules/database.js');
+var testUserRequest = require('./testUserRequest.json');
+var User = require('../models/User.js').UserTest;
+
+var database = new DataBase("test");
+
+
+describe('Array', function() {
```

Comment:
Can we get rid of this test since we have other tests to confirm that the test suite is running?

Reply:
Sure, I left it in there as a reference but we could always look in Github for it again if we want to.